TikiClient/Classes/TikiClient.swift

```diff
@@ -14,12 +14,14 @@ import SwiftUI
 /// `TikiClient` is the top-level entry point for the TIKI Client Library. It offers simple methods
 /// that call the underlying libraries to perform common operations. Programmers can use it to
 /// simplify the integration process or opt for individual libraries based on their specific needs.
+
 public class TikiClient {
     
     public static let auth = AuthService()
     public static let capture = CaptureService()
     public static let license = License()
     public static let tracking = Tracking()
+    public static let location =  ()
```

Comment:
This should not be here

Reply:
Was an mistake, i erase the reference without noticing.